lxd/networks.go

```diff
@@ -23,6 +23,7 @@ import (
 	"github.com/lxc/lxd/lxd/db"
 	"github.com/lxc/lxd/lxd/device"
 	"github.com/lxc/lxd/lxd/dnsmasq"
+	"github.com/lxc/lxd/lxd/instance"
```

Comment:
This change should get rebased into whatever commit needs the extra import.

Reply:
I'm not sure what you mean by this, I believe the usage prior to our changes was importing instanceType, but that was removed when we rebased back on to master so I added the import to build. The base code uses instance and we don't in any of our changes.

Reply:
![image](https://user-images.githubusercontent.com/26911504/70549786-6a147480-1b3a-11ea-9136-aea17efecf65.png)

Reply:
I could fold it into our first commit, but since it didn't directly involve our code to my understanding I figured it would be better to keep it out of our changes

Reply:
Commented in the earlier commit where you mistakenly remove that import from `networks.go`.